amd/amdgpu/Makefile

```diff
@@ -435,21 +446,8 @@ SRCS+=	device_if.h vnode_if.h bus_if.h pci_if.h device_if.h iicbus_if.h opt_drm.
 
 .include <bsd.kmod.mk>
 
-CFLAGS.gcc+= -Wno-redundant-decls -Wno-cast-qual -Wno-unused-but-set-variable
-.if ${MACHINE_CPUARCH} == "powerpc"
```

Comment:
Why do you remove all the powerpc stuff?

Reply:
I've explained this in the commit message. This part of the code (DML) is not used on powerpc because https://github.com/FreeBSDDesktop/kms-drm/blob/6e6a861de0b6c519c23caca927eca9161931caaf/drivers/gpu/drm/drm_os_config.h#L127-L129

Upstream only builds it on amd64 https://github.com/torvalds/linux/commit/bf2e2e2e0ea9ace6ed11423497664656943ec55f

IIUC, this code is only for Raven Ridge laptops